spec/importc.dd

```diff
@@ -837,6 +837,49 @@ $(H2 $(LNAME2 wrapping, Wrapping C Code))
     $(P Then, `import file_ic;` instead of `import file;`, and use `S_t` when $(CCODE struct S) is desired.)
 
 
+$(H2 $(LNAME2 ctod, Converting C Code to D Code))
+
+    $(P Sometimes its desirable to go further than importing C code, to actually do a C source to
+    D source conversion. Reasons include:)
+
+    $(UL
+    $(LI Migrating a C project to a D project.)
+    $(LI Equivalent D code can compile much faster, due to not needing a preprocessor, etc.)
+    $(LI Tweaking the D code to add attributes for memory safety, purity, etc.)
+    $(LI Eliminate the need for C-isms in the D part of the project.)
+    )
+
+    $(P This can be done with the D compiler by using the $(CONSOLE -Hf) switch:)
```

Comment:
Using `CONSOLE` breaks the sentence in 2 but also causes an empty paragraph number underneath. Also it's not a complete shell command anyway.
```suggestion
    $(P This can be done with the D compiler by using the $(D -Hf) switch:)
```